shielder/deploy/deploy.sh

```diff
@@ -9,7 +9,7 @@ SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
 export NODE_IMAGE="public.ecr.aws/p6e8q1z1/aleph-node-liminal:d93048e"
 export CLIAIN_IMAGE="public.ecr.aws/p6e8q1z1/cliain-liminal:d93048e"
-export CARGO_IMAGE="public.ecr.aws/p6e8q1z1/ink-dev:1.0.0"
+export CARGO_IMAGE="public.ecr.aws/p6e8q1z1/ink-dev:1.1.0"
```

Comment:
since you are using it as `docker_shell` you might want to rename image; up to you (I don't have a good proposition :P )

Reply:
Went with `INK_DEV_IMAGE` and change to function name to `docker_ink_dev` to be more consistent.